src/CoreMedia/CMFormatDescription.cs

```diff
@@ -489,13 +489,13 @@ public bool VideoMatchesImageBuffer (CVImageBuffer imageBuffer)
 		public static CMVideoFormatDescription? FromHevcParameterSets (List<byte[]> parameterSets, int nalUnitHeaderLength, NSDictionary extensions, out CMFormatDescriptionError error)
 		{
 			if (parameterSets is null)
-				throw new ArgumentNullException (nameof (parameterSets));
+				ObjCRuntime.ThrowHelper.ThrowArgumentNullException (nameof (parameterSets));
 
 			if (parameterSets.Count < 3)
-				throw new ArgumentException ($"{nameof (parameterSets)} must contain at least three elements");
+				ObjCRuntime.ThrowHelper.ThrowArgumentException ("must contain at least three elements", nameof (parameterSets));
```

Comment:
Argument name comes first, then the message:

```suggestion
				ObjCRuntime.ThrowHelper.ThrowArgumentException (nameof (parameterSets), "must contain at least three elements");
```